src/components/NavBar.jsx

```diff
@@ -0,0 +1,15 @@
+const React = require("react");
+const Link = require("react-router").Link;
+
+const NavBar = React.createClass({
+	render: function(){
+		return (
+			<ul>
+				<li><Link to='/'>Home</Link><Link to='/about'>About</Link></li>
+				
+			</ul>
+		);
+	}
+});
+
+module.exports = NavBar;
```

Comment:
Nice navbar. It's not visible anywhere, so maybe try requiring and adding this component to `PersonShow.jsx` and `Yearbook.jsx`